src/Spice86.Core/CLI/CommandLineParser.cs

```diff
@@ -9,19 +9,34 @@
 using System;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Reflection;
 
 
 /// <inheritdoc cref="ICommandLineParser" />
 public class CommandLineParser : ICommandLineParser {
     /// <inheritdoc />
     public Configuration ParseCommandLine(string[] args) {
-        ParserResult<Configuration> result = Parser.Default.ParseArguments<Configuration>(args);
+        int indexOfPassedExeArgs = Array.IndexOf(args, GetCommandLineName<Configuration>(nameof(Configuration.ExeArgs), false));
```

Comment:
How about moving that to a method like ExtractDosExeArgs so that it's more obvious what this does?

Reply:
or a slightly simpler method:
```csharp
    private static List<string> ProcessArgs(string[] args) {
        var processedArgs = new List<string>();
        for (int i = 0; i < args.Length; i++) {
            if (IsExeArg(args[i]) && i + 1 < args.Length) {
                processedArgs.Add(args[i]);
                processedArgs.Add(args[i + 1]);
                i++;
            } else {
                processedArgs.Add(args[i]);
            }
        }

        return processedArgs;
    }
```

Reply:
Done.